content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,70 @@ positional arguments:
+For example:
 
 ```yaml
 stages:
   learn:
-    cmd: ./deep.py
+    cmd: python deep.py
     params:
-      - epochs # track specific parameter (from params.yaml)
-      - tuning.learning-rate
-      - myparams.toml: # track specific params from custom file
-          - batch_size
-      - config.json: # track all parameters in this file
+      - epochs # specific param from params.yaml
+      - tuning.learning-rate # from params.yaml
+      - myparams.toml:
+          - batch_size # specific param from custom file
+      - config.json: # all params in this file
 ```
 
-In contrast to a regular <abbr>dependency</abbr>, a parameter dependency is not
-a file or directory. Instead, it consists of a _parameter name_ (or key) in a
-_parameters file_, where the _parameter value_ should be found. This allows you
-to define [stage](/doc/command-reference/run) dependencies more granularly:
-changes to other parts of the params file will not affect the stage. Parameter
-dependencies also prevent situations where several stages share a regular
-dependency (e.g. a config file), and any change in it invalidates all of them
-(see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
-
-The default **parameters file** name is `params.yaml`, but any other YAML 1.2,
-JSON, TOML 1.0, or [Python](#examples-python-parameters-file) files can be used
-additionally (listed under `params:` as shown in the sample above). These files
-are typically written manually (or they can be generated) and they can be
-versioned directly with Git.
-
-**Parameter values** should be organized in tree-like hierarchies (dictionaries)
-inside params files (see [Examples](#examples)). DVC will interpret param names
-as the tree path to find those values. Supported types are: string, integer,
-float, boolean, and arrays (groups of params). Note that DVC does not ascribe
-any specific meaning to these values.
+<admon type="info">
 
-DVC saves parameter names and values to `dvc.lock` in order to track them over
-time. They will be compared to the latest params files to determine if the stage
-is outdated upon `dvc repro` (or `dvc status`).
+See [more details] about this syntax.
 
-> Note that DVC does not pass the parameter values to stage commands. The
-> commands executed by DVC will have to load and parse the parameters file by
-> itself.
+</admon>
 
 The `dvc params diff` command is available to show parameter changes, displaying
 their current and previous values.
 
-💡 Parameters can also be used for
-[templating](/doc/user-guide/project-structure/dvcyaml-files#templating)
-`dvc.yaml` itself.
+<abbr type="tip">
+
+Parameters can also be used for [templating] `dvc.yaml` itself.
+
+</abbr>
+
+DVC saves parameter names and values to `dvc.lock` in order to track them over
+time. They will be compared to the latest params files to determine if the stage
+is outdated upon `dvc repro` (or `dvc status`).
+
+DVC does not pass the parameter values to [stage commands]. The commands
```

Comment:
DVC can pass the values to commands now: see https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#dict-unpacking. Seems worth mentioning here.

Reply:
Hmmmm right. OK I need to change a few things...
Done (4005a8a13)! PTAL